src/clcontext.h

```diff
@@ -41,6 +41,7 @@ typedef struct {
     cl_ulong max_memory;
     size_t num_work_group;
     KernelInfo kernel_info;
+    uint64_t hashTimes;
```

Comment:
Avoid [camel case](https://en.wikipedia.org/wiki/Camel_case).

Reply:
> getHashTimes() shoulde also be included in the test case.

Yes, it would be included in the future pull request of getting the hash rate.